machine_test.go

```diff
@@ -396,7 +394,6 @@ func TestStartVMMOnce(t *testing.T) {
 	if err != nil {
 		t.Fatalf("unexpected error: %v", err)
 	}
-	defer m.StopVMM()
```

Comment:
What's the rationale for removing this?

Reply:
Calling `NewMachine` won't start the `VMM`, so no need to call `StopVMM`.